input/fsh/ValueSet-DiseaseCodes.fsh

```diff
@@ -1,8 +1,10 @@
 ValueSet: DiseaseCodes
 Id: disease-codes
 Title: "Disease Codes"
-Description: "This value set includes all codes from NCIt (NCI Thesaurus)."
+Description: "This value set includes all codes from this FHIR server's representation of OMIM, Mondo and NICt CodeSystems."
 * ^version = "0.1.0"
 * ^status = #draft
 * ^experimental = false
+* include codes from system $omim
+* include codes from system $mondo
 * include codes from system $ncit.owl
```

Comment:
In the Alias file, I don't see `$ncit.owl`? And, we won't suffix `owl`, right?

Reply:
It's actually line number 18 in the Alias file and has been there since the original release. If we want to change it to remove the .owl, I can do that.

Reply:
Like this one (and probably the only one): https://github.com/ncpi-fhir/ncpi-fhir-ig/blob/main/input/fsh/Instance-Disease.fsh#L14

Reply:
done